jormungandr-lib/src/interfaces/block0_configuration/active_slot_coefficient.rs

```diff
@@ -8,7 +8,7 @@ use std::{convert::TryFrom, fmt, str::FromStr as _};
 use thiserror::Error;
 
 #[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
-pub struct ActiveSlotCoefficient(pub(crate) Milli);
+pub struct ActiveSlotCoefficient(Milli);
```

Comment:
Are you sure ?
It will correspond to the another 'interface' types:
`pub struct BlockContentMaxSize(fragment::BlockContentSize);`
`pub struct EpochStabilityDepth(u32);`
`pub struct NumberOfSlotsPerEpoch(u32);` etc

Reply:
Not sure I understand your concern, can you elaborate a bit?

Reply:
In my opinion we should follow the same code style for all interface/wrapper types tike `BlockContentMaxSize`, `EpochStabilityDepth`, `NumberOfSlotsPerEpoch`, `NumberOfSlotsPerEpoch`, `ActiveSlotCoefficient` etc.  So if for example `pub struct BlockContentMaxSize(fragment::BlockContentSize);` has private visibility for `fragment::BlockContentSize` I think we should follow the same for the another structs as `ActiveSlotCoefficient`.
And generally if the usage of this values comes only on the library side, so I think it is better to hide the internal type and make it immutable.

Reply:
forgive me, I read the reversed diff 😅  All good!